pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -131,6 +131,12 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 			ObjectMeta: metav1.ObjectMeta{
 				Name: ClusterReaderRoleName,
 			},
+			AggregationRule: &rbacv1.AggregationRule{
+				ClusterRoleSelectors: []metav1.LabelSelector{{MatchLabels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-cluster-reader": "true"}}},
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: AggregatedClusterReaderRoleName, Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-cluster-reader": "true"}},
```

Comment:
you want `"rbac.authorization.k8s.io/aggregate-to-view"` too right?  Otherwise nothing from upstream or the community will aggregate the way you want.

Reply:
These are the current rules to initially aggregate to cluster-reader, why aggregate them to view also? My intent was to assemble cluster-reader by aggregating the current rules plus view into cluster-reader.

Reply:
Sorry commented on the wrong line.  the selector needs that.  I may have been in the wrong spot in the file.

Reply:
No problem, I thought that may have been the case, so I tried updating the cluster-reader role's MatchLabel with both aggregate-to-cluster-reader and aggregate-to-view labels, but nothing gets aggregated. I think that matching on two labels means a to-aggregate role must have both labels.

Reply:
> No problem, I thought that may have been the case, so I tried updating the cluster-reader role's MatchLabel with both aggregate-to-cluster-reader and aggregate-to-view labels, but nothing gets aggregated. I think that matching on two labels means a to-aggregate role must have both labels.

I wrote it to OR them.  Try adding a second label selector instead of adding to the first selector